tests/js/task-resolves/symbol.js

```diff
@@ -0,0 +1,4 @@
+//# starling-not-a-test
+async function main() {
+  return new Symbol("I do not structured clone");
```

Comment:
I wonder if symbols created using `Symbol.for` do structured-clone. They should, IIUC, but I might well be mistaken. Would be good to test either way.